girderformindlogger/api/v1/applet.py

```diff
@@ -606,59 +611,53 @@ def inviteUser(self, applet, role="user", email='', displayName='', MRN=''):
                 "Only coordinators and managers can invite users."
             )
 
-        if not invitedUser:
+        if role not in USER_ROLE_KEYS:
             raise ValidationException(
-                'the user with such email does not exist', 'email'
+                'Invalid role.',
+                'role'
             )
-        try:
-            if role not in USER_ROLE_KEYS:
-                raise ValidationException(
-                    'Invalid role.',
-                    'role'
-                )
 
-            invitation = Invitation().createInvitationForSpecifiedUser(
-                applet = applet,
-                coordinator = thisUser,
-                role = role,
-                user = invitedUser,
-                displayName = displayName,
-                MRN = MRN,
```

Comment:
> @jj105 `it seems value type of level field in dev database is updated and issue was happening on it`
> What collection in?
> because i didn't see any error

I think it was in user collection.

Reply:
> @jj105 why do you put spaces on both sides of the equal sign?
> in all places it looks different. i think it should be merged. just my opinion

in the future, we could have cases that insert parameters between parameters we have at the moment.
we won't need to worry much about existing function calls if we insert parameters like this.

Reply:
> > @jj105 why do you put spaces on both sides of the equal sign?
> > in all places it looks different. i think it should be merged. just my opinion
> 
> in the future, we could have cases that insert parameters between parameters we have at the moment.
> we won't need to worry much about existing function calls if we insert parameters like this.

Why do we need it?
method's parameters should contain only parameters or variables. It's necessary to use any conditions or something else

Reply:
so, what's your suggestion?
what should it be do you think?

Reply:
this parameters should be merged like
```
applet=applet,
coordinator=thisUser,
role=role,
user=invitedUser,
displayName=displayName,
MRN=MRN,
```

that's more correct

Reply:
this is just suggestion to write in better code style. do not worry about that for now. as I see, if we have to change this, many things should be changed/updated

Reply:
this is just suggestion to write in better code style. do not worry about that for now. as I see, if we have to change this, many things should be changed/updated

Reply:
I made one more commit to update it.